assets/js/account/account-order-detail.tpl.liquid

```diff
@@ -0,0 +1,130 @@
+<div class="page-header">
+    <h3>
+        <a ng-href="{{ 'account/b4#/orders/' | absolute_url }}?pageNumber={% raw %}{{$ctrl.pageNumber}}{% endraw %}" class="btn btn-primary btn-alt btn-xs btn-return">
+            <span class="fa fa-arrow-left"></span> {{ 'customer.orders.return' | t }}
+        </a>
+    </h3>
+</div>
+<div id="order-detail" class="row">
+    <div class="col-md-9">
+        <div class="row">
+            <div class="col-md-12">
+                <div class="panel panel-default">
+                    <div class="panel-heading">
```

Comment:
Panels is not exists in Bootstrap 4, use cards instead of remove if not needed these styles here

Reply:
@asvishnyakov it's our old page, not migrated one

Reply:
Refactoring